include/odp/api/spec/proto_stats.h

```diff
@@ -0,0 +1,163 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright(C) 2021 Marvell.
+ */
+
+/**
+ * @file
+ *
+ * ODP Proto Stats
+ */
+
+#ifndef ODP_API_SPEC_PROTO_STATS_H_
+#define ODP_API_SPEC_PROTO_STATS_H_
+#include <odp/visibility_begin.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @addtogroup odp_proto_stats
+ *  @{
+ */
+
+/**
+ * @def ODP_PROTO_STATS_INVALID
+ * Invalid proto stats handle
+ */
+
+/** ODP proto stats id */
+typedef enum odp_proto_stats_id_t {
+	/** Max */
+	ODP_PROTO_STATS_ID_MAX,
+} odp_proto_stats_id_t;
+
+/** ODP proto stats params */
+typedef struct odp_proto_stats_param_t {
+	/** Stats id mask
+	 *
+	 * Each bit corresponds to one ODP_PROTO_STATS_ID_* and
+	 * bit position is same as ODP_PROTO_STATS_ID_*.
+	 */
+	uint64_t stats_mask;
+} odp_proto_stats_param_t;
```

Comment:
This should say that the bits set in stats_mask define which statistics are enabled. Now it explains which bit is which but. not what the bits do.